Makefile

```
@@ -317,8 +329,11 @@ sync-helm-multi-resource-helpers:
 		echo "  ✓ Copied to charts/$$chart/templates/_resources.tpl"; \
 	done
 
+boilerplate:
+	hack/boilerplate.sh
+
 # This runs all necessary steps to prepare for a commit.
-precommit: ensure-go-version-upgrade sync-deps sync-img-env vet go-lint py-fmt py-lint generate tidy manifests uv-lock generate-quick-install-scripts generate-chart-manifests sync-helm-common-helpers sync-helm-common-resource-helpers sync-helm-multi-resource-helpers verify-pinned-actions verify-minimal-crd-sync
+precommit: ensure-go-version-upgrade sync-deps sync-img-env vet go-lint py-fmt py-lint generate tidy manifests uv-lock generate-quick-install-scripts generate-chart-manifests sync-helm-common-helpers sync-helm-common-resource-helpers sync-helm-multi-resource-helpers verify-pinned-actions verify-minimal-crd-sync boilerplate
```

checkmake (0.3.2)

[warning] Target "precommit" should be declared PHONY.

(phonydeclared)